src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,444 @@
+
+#### Image Acquisition
+
+{{ MACROS___make_metadata_table(
+   {
+      "PixelSize": "REQUIRED",
+      "PixelSizeUnits": "REQUIRED",
+      "Immersion": "OPTIONAL",
+      "NumericalAperture": "OPTIONAL",
+      "Magnification": "OPTIONAL",
+      "ImageAcquisitionProtocol": "OPTIONAL",
+      "OtherAcquisitionParameters": "OPTIONAL",
+   }
+) }}
+
+#### Sample
+
+{{ MACROS___make_metadata_table(
+   {
+      "BodyPart": ("RECOMMENDED", "From [DICOM Body Part Examined](http://dicom.nema.org/medical/dicom/current/output/chtml/part16/chapter_L.html#chapter_L) (for example `\"BRAIN\"`)."),
+      "BodyPartDetails": "RECOMMENDED",
+      "BodyPartDetailsOntology": "OPTIONAL",
+      "SampleEnvironment": "RECOMMENDED",
+      "SampleEmbedding": "OPTIONAL",
+      "SampleFixation": "OPTIONAL",
+      "SampleStaining": "RECOMMENDED",
+      "SamplePrimaryAntibody": "RECOMMENDED",
+      "SampleSecondaryAntibody": "RECOMMENDED",
+      "SliceThickness": "OPTIONAL",
+      "TissueDeformationScaling": "OPTIONAL",
+      "SampleExtractionProtocol": "OPTIONAL",
+      "SampleExtractionInstitution": "OPTIONAL",
+   }
+) }}
+
+#### Chunk Transformations
+
+Chunk transformations metadata describes the spatial relation between chunks
+of the same sample in an implicit coordinate system.
+
+-   The source frame of reference is the frame of reference of the associated image.
+
+-   The target frame of reference is the implicit coordinate system of the transform.
+
+-   The target frame of reference has the same units as the `PixelSizeUnits` metadata.
+
+-   The chunk transformation is described by 2 metadata fields: an affine transformation matrix
+    and a description of the axis of the matrix.
+
+-   Other transformations should be described in derivatives.
+
+{{ MACROS___make_metadata_table(
+   {
+      "ChunkTransformationMatrix": "RECOMMENDED if `<chunk-index>` is used in filenames",
+      "ChunkTransformationMatrixAxis": "REQUIRED if `ChunkTransformationMatrix` is present",
+   }
+) }}
+
+An example of chunk transformations JSON metadata for `chunk-01` and `chunk-05` of Figure 2
+is shown below:
+
+![Figure 2](images/microscopy_chunks2.png "Example figure for chunks transformations")
+
+Figure 2: Example figure for chunks transformations.
+
+In this example, there is no scaling and `chunk-01` is at the origin.
+`chunk-05` is translated by 5 um along `X+` and by 3 um along `Y+`.
+
+**`*_chunk-01_<suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 0],
+                                      [0, 1, 0],
+                                      [0, 0, 1]],
+        "ChunkTransformationMatrixAxis": ["X", "Y"]
+}
+```
+
+**`*_chunk-05_<suffix>.json`**:
+```JSON
+{
+        "PixelSize": [1, 1],
+        "PixelSizeUnits": "um",
+        "ChunkTransformationMatrix": [[1, 0, 5],
+                                      [0, 1, 3],
+                                      [0, 0, 1]],
```

Comment:
I wonder what this last row of the matrix is representing? 
If axis are X and Y (e.g. in this example), I see why there must be 3 columns, one for scaling in X, one for scaling in Y and one for traslation. But I wonder why 3 rows and not just 2? Can you think of an example where the very last row is different from [0, 0, 1]?
Sorry for bugging you so much about the Chuck transformations, I'm new to this concept and I want to ensure I understand it properly. Thanks!

Reply:
No problem!
If I'm not mistaken, by definition the last row of an affine matrix will always be [0, 0, 1] for 2D and [0, 0, 0, 1] for 3D.
This line allows the translation to be done with matrix multiplication.
A full example can be found here: https://en.wikipedia.org/wiki/Transformation_matrix#Affine_transformations